src/status_im/ui/screens/chat/components/input.cljs

```diff
@@ -587,36 +588,38 @@
                                       (reanimated/set-shared-value bg-opacity (reanimated/with-timing 0)))
                                     (reanimated/set-shared-value translate-y (reanimated/with-timing (- y)))
                                     (reanimated/set-shared-value shared-height (reanimated/with-timing (min y max-height)))))
-              [reanimated/view {:style (reanimated/apply-animations-to-style
-                                        {:height shared-height}
-                                        {})}
+              [react/view
```

Comment:
Maybe use fragment (`[:<>` ) instead of view

Reply:
Thanks @briansztamfater . Fixed it.